pkg/urlmanager/urlmanager.go

```diff
@@ -9,27 +9,17 @@ import (
 	"github.com/LambdaTest/synapse/pkg/global"
 )
 
-// GetDownloadURL returns file download url for given git provider
-func GetDownloadURL(gitprovider, repoSlug, commitID, fileName string) (string, error) {
```

Comment:
why this is removed ? How are we download raw files ?

Reply:
We are not using it anywhere

Reply:
https://github.com/LambdaTest/test-at-scale/pull/47#discussion_r822497660